go-controller/go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	k8s.io/client-go v0.20.0
 	k8s.io/klog v0.0.0-20181102134211-b9b56d5dfc92
 	k8s.io/klog/v2 v2.4.0
+	k8s.io/kubernetes v0.0.0-00010101000000-000000000000
```

Comment:
hey @tssurya this shows that we're vendoring k/k for the first time here...

Reply:
oh heh you are right, https://github.com/ovn-org/ovn-kubernetes/tree/master/go-controller/vendor/k8s.io sorry. So I shouldn't do it this way?

Reply:
@tssurya I think what Casey was trying to say is that it's not good practice to vendor K8s/k8s, and basically we shouldn't do it..... Instead I think we should just copy the k8s.io/kubernetes/pkg/util/taints package into our codebase manually.  I.e Add a new folder in /pkg/util and put the taints code there

Reply:
(But don't quote me on it :) )

Reply:
yea we discussed this in the meeting. Considering I need to copy a lot of dependent functions, decision as to move forward with vendoring.

Reply:
You don't _have to_ use `pkg/util/taints`. Taints are part of the public API of v1.Node; as long as you are setting/clearing them correctly, you can do it "by hand". The utility functions are just there to make things easier, but if you end up having to vendor in all of k/k, that's not really "easier" any more.